samples/stock/demo/area/demo.js

```diff
@@ -1,53 +1,232 @@
+                style: {
+                    color: '#ffffff',
+                    opacity: 1,
+                    textOutline: '#000000'
                 }
             }
         },
+        maskFill: 'rgba(181, 145, 143, 0.2)'
+    },
 
-        series: [{
-            name: 'AAPL Stock Price',
-            data: data,
-            type: 'area',
-            threshold: null,
-            tooltip: {
-                valueDecimals: 2
+    exporting: {
+        buttons: {
+            contextButton: {
+                symbolStroke: '#d9d7d7',
+                theme: {
+                    fill: '#000000',
+                    states: {
+                        hover: {
+                            fill: '#000000'
+                        },
+                        select: {
+                            fill: '#f23644'
+                        }
+                    }
+                }
+            }
+        }
+    },
+
+    scrollbar: {
+        barBackgroundColor: '#5f5959',
+        trackBorderColor: '#5f5959'
+    },
+
+    rangeSelector: {
+        buttonTheme: {
+            fill: 'none',
+            stroke: 'none',
+            'stroke-width': 0,
+            r: 8,
+            style: {
+                color: '#d9d7d7',
+                fontWeight: 'bold',
+                fontSize: '1em'
             },
-            fillColor: {
-                linearGradient: {
-                    x1: 0,
-                    y1: 0,
-                    x2: 0,
-                    y2: 1
+            states: {
+                select: {
+                    fill: '#f23644',
+                    style: {
+                        color: '#ffffff'
+                    }
                 },
+                hover: {
+                    style: {
+                        color: '#000000'
+                    }
+                }
+            }
+        }
+    },
+
+    plotOptions: {
+        series: {
+            threshold: null,
+            color: '#f23644',
+            fillColor: {
+                linearGradient: [0, 0, 0, 450],
                 stops: [
-                    [0, Highcharts.getOptions().colors[0]],
-                    [
-                        1,
-                        Highcharts.color(
-                            Highcharts.getOptions().colors[0]
-                        ).setOpacity(0).get('rgba')
-                    ]
+                    [0, '#5c0d13'],
+                    [1, '#000000']
                 ]
             }
+        }
+    },
+
+    tooltip: {
+        backgroundColor: '#212020',
+        style: {
+            color: '#ffffff'
+        }
+    }
+});
+
+// Configuring the chart.
+Highcharts.stockChart('container', {
+
+    title: {
+        text: 'BTCETH',
+        align: 'left',
+        y: 30
+    },
+
+    xAxis: {
+        lineWidth: 0,
+        crosshair: {
+            width: 1,
+            color: '#616161',
+            zIndex: 3
+        }
+    },
+
+    scrollbar: {
+        barBorderRadius: 8,
```

Comment:
```suggestion
        barBorderRadius: 4,
```
The border radius should be no more than half of the height, otherwise it gets this elongated shape that looks like a bug